crates/bevy_ecs/src/reflect.rs

```diff
@@ -62,6 +97,16 @@ impl ReflectComponent {
         (self.reflect_component_mut)(world, entity)
     }
 
+    /// Directly copies the value of the component of type `C` to a new entity
+    ///
+    /// This method creates a new component of type `C` using the [FromWorld] trait,
+    /// sets its value to the value of the component of type `C` on the `source_entity`,
+    /// and then inserts the new component into the `destination_entity`.
+    ///
+    /// WARNING: this method does not use the `Clone` method: instead, memory is copied directly.
+    /// This can have unexpected negative consequences if you are relying on ref-counting or the like.
+    ///
+    /// PANICS: the `source_entity` in the `source_world` must have a component of type `C`
```

Comment:
Use the conventional `# Panics` markup here.

Reply:
```suggestion
    /// # Panics
    /// The `source_entity` in the `source_world` must have a component of type `C`
```